docs/about/release_process.rst

```diff
@@ -8,10 +8,13 @@ Release Process
 
    uv lock
 
-#. Update the release notes in about/whats_new.rst and the fallback version number in pyproject.toml in the
+#. Create a new **Tag** and **Draft Release** using the `GitHub Releases Web UI`_
+
+#. Based on the generated release notes from GitHub, update the release notes in
+   about/whats_new.rst and the fallback version number in pyproject.toml in the
    ``develop`` branch via a PR.
 
-#. Create a new **Tag** and **Release** using the `GitHub Releases Web UI`_
+#. Change the Draft Release to Published using the `GitHub Releases Web UI`_
```

Comment:
@omad Isn't the tag going to be pointing before the updated whats_new.rst at this point?

I don't think the draft release makes a git tag in the repository, that happens when publishing the release, so there should be time to still change where the candidate tag is pointing at this stage, so the updated pyproject.toml and whats_new.rst could be included.